packages/integration-sdk-testing/src/jest.ts

```diff
@@ -286,6 +286,45 @@ function dedupSchemaEnumValues(schema: GraphObjectSchema): GraphObjectSchema {
   };
 }
 
+/**
+ * Step to remove specified properties marked for exclusion
+ * in the schemas. This step will run after all deduplication
+ * processes to ensure that the properties will be removed from
+ * the final schema.
+ *
+ * Example: parameter: { exclude: true }
+ *
+ * @param schema
+ */
+function removeSchemaExcludedValues(
+  schema: GraphObjectSchema,
+): GraphObjectSchema {
+  if (!schema.properties) {
+    return schema;
+  }
+
+  const newProperties: Record<string, any> = {};
+
+  for (const propertyName in schema.properties) {
+    const property = schema.properties[propertyName];
```

Comment:
couldn't you do `for (const [propertyName, property] of Object.entries(schema.properties)` above so you don't need this line?

Reply:
Yes but I copied the implementation from the other schema modifying functions and wanted to be consistent.